matchmaking/frontend/index.html

```diff
@@ -6,7 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Vite + React</title>
   </head>
-  <body>
+  <body style="margin: 0;">
```

Comment:
This inline style can be removed and, if necessary, added to the new base.css file that was added in my PR from earlier today.